### ISSUE TITLE
Cats: Stream & Sink

### ISSUE THREAD

**Issue by regiskuckaertz** [zio/zio#1006]
`Alternative`, `Monad`, `Arrow`, etc.

Where should we put them @iravid ?

**Comment by iravid** [comment]
Ugh. The choice is between creating `interop-cats-stream` and making the `interop-cats` depend on the `stream` artifact, right?

**Comment by regiskuckaertz** [comment]
yeah 🤕 is it safe to use `optional` or `provided` if going in the second direction in that case?

**Comment by iravid** [comment]
I think so. Let's ask @kaishh to make sure?

**Comment by ghostdogpr** [comment]
@jdegoes shall we transfer issues like this one to the cats interop repository, or do you prefer having everything here?
